chain/sync.go

```diff
@@ -538,18 +538,25 @@ func (syncer *Syncer) ValidateBlock(ctx context.Context, b *types.FullBlock) err
 			return xerrors.Errorf("received block was from slashed or invalid miner")
 		}
 
-		mpow, tpow, err := stmgr.GetPower(ctx, syncer.sm, baseTs, h.Miner)
+		_, tpow, err := stmgr.GetPower(ctx, syncer.sm, baseTs, h.Miner)
 		if err != nil {
 			return xerrors.Errorf("failed getting power: %w", err)
 		}
 
+		pset, err := stmgr.GetMinerProvingSet(ctx, syncer.sm, baseTs, h.Miner)
+		if err != nil {
+			return xerrors.Errorf("failed getting proving set: %w", err)
+		}
+		for _, s := range pset {
+			if s.Info.Info.RegisteredProof == 0 {
```

Comment:
can probably remove this sanity check loop

Reply:
removed